### ISSUE TITLE
Node.js and function form of 'use strict'

### ISSUE THREAD

**Issue by Sannis** [douglascrockford/JSLint#124]
As I know, function form of 'use strict' was proposed to avoid global variable leaks. But this is useless for Node.js modules because they don't share global context. So I propose to disable this warning when `node: true` and `sloppy: true`:

```
Problem at line 1 character 1: Use the function form of 'use strict'.
```
